modules/local/pyconvertosim/main.nf

```diff
@@ -0,0 +1,72 @@
+process PYCONVERTOSIM {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/biopython:1.70--np112py27_0':
+        'biocontainers/biopython:1.70--np112py27_0' }"
+
+    input:
+    tuple val(meta), path(vcf_benign)
```

Comment:
here I'd say either we pass the meta on both or none. either would be ok

Reply:
I would probably keep the meta on both so we can move along variables like chromosome and targets